webapp/javascript/components/FileUploader.tsx

```diff
@@ -30,8 +30,14 @@ export default function FileUploader({ onUpload }: Props) {
           const s = JSON.parse(
             String.fromCharCode.apply(null, new Uint8Array(binaryStr))
           );
-          setFile({ file, ...s });
-          onUpload({ file, ...s });
+          const { flamebearer } = s;
+          const calculatedLevels = deltaDiffWrapper(
+            flamebearer.format,
+            flamebearer.levels
+          );
```

Comment:
Ugh I gate this deltaDiffWrapper`, this is more of a note to self, I think sometime in the near future we shall unify this initial processing. Or move to the backend.